test/sagemaker_tests/huggingface_pytorch/training/conftest.py

```diff
@@ -0,0 +1,249 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import logging
+import platform
+import shutil
+import sys
+import tempfile
+
+import pytest
+import boto3
+
+from sagemaker import LocalSession, Session
+from sagemaker.pytorch import PyTorch
+
+from .utils import image_utils, get_ecr_registry
+
+
+logger = logging.getLogger(__name__)
+logging.getLogger('boto').setLevel(logging.INFO)
+logging.getLogger('boto3').setLevel(logging.INFO)
+logging.getLogger('botocore').setLevel(logging.INFO)
+logging.getLogger('factory.py').setLevel(logging.INFO)
+logging.getLogger('auth.py').setLevel(logging.INFO)
+logging.getLogger('connectionpool.py').setLevel(logging.INFO)
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+NO_P2_REGIONS = [
+    "ap-east-1",
+    "ap-northeast-3",
+    "ap-southeast-2",
+    "ca-central-1",
+    "eu-central-1",
+    "eu-north-1",
+    "eu-west-2",
+    "eu-west-3",
+    "us-west-1",
+    "sa-east-1",
+    "me-south-1",
+    "cn-northwest-1",
+    "eu-south-1",
+    "af-south-1",
+]
+NO_P3_REGIONS = [
+    "ap-east-1",
+    "ap-northeast-3",
+    "ap-southeast-1",
+    "ap-southeast-2",
+    "ap-south-1",
+    "ca-central-1",
+    "eu-central-1",
+    "eu-north-1",
+    "eu-west-2",
+    "eu-west-3",
+    "sa-east-1",
+    "us-west-1",
+    "me-south-1",
+    "cn-northwest-1",
+    "eu-south-1",
+    "af-south-1",
+]
+
+
+def pytest_addoption(parser):
+    parser.addoption('--build-image', '-D', action='store_true')
+    parser.addoption('--build-base-image', '-B', action='store_true')
+    parser.addoption('--aws-id')
+    parser.addoption('--instance-type')
+    parser.addoption('--docker-base-name', default='pytorch')
+    parser.addoption('--region', default='us-west-2')
+    parser.addoption('--framework-version', default='')
```

Comment:
I suppose we also need "--transformers-version"

Reply:
we don't need that.. it gets derived from image_uri in SM estimator